deploy/controller-infra/base/deploy.yaml

```diff
@@ -145,6 +145,29 @@ spec:
             requests:
               memory: 20Mi
               cpu: 10m
+        - name: csi-resizer
+          image: registry.k8s.io/sig-storage/csi-resizer:v1.13.1
+          args:
+            - "-csi-address=/csi/csi.sock"
+            - "-kubeconfig=/var/run/secrets/tenantcluster/value"
+            - "-v=5"
+            - "-timeout=3m"
+            - '-handle-volume-inuse-error=false'
+          volumeMounts:
+            - name: socket-dir
+              mountPath: /csi
+            - name: tenantcluster
+              mountPath: "/var/run/secrets/tenantcluster"
+          resources:
+            limits:
+              memory: 400Mi
```

Comment:
Any reason for this limit here, in general I know we want limits, but since this thing is watching a large number of resource in the cluster it can potentially grow large in large clusters. That is why for control plane pods we usually just set the requests not the limits.

Reply:
No reason, fair point, removed